project.toml

```diff
@@ -0,0 +1,40 @@
+[build-system]
+requires = ["setuptools>=68.1.2",
+            "setuptools_scm[toml]>=6.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "GemGIS"
+authors = [
+    {name = "Alexander Jüstel", email = "alexander.juestel@rwth-aachen.de"},
+]
```

Comment:
there is an authors key and a maintainers key. but it would be good to migrate all of your authors over

Arthur Endlein Correia, Florian Wellmann, Marius Pischke ... 
it is ok if you don't have everyone's emails you can either skip the email key totally or just skip those without emails. i've found it easiest to just skip emails if you don't have everyone. Or you could make a maintainer list with just names too the format is the same as authors. 


```
maintainers = [
    {name = "Alexander Jüstel"},
    {name= "Arthur Endlein Correia"},
     {another name here...}
]
```

Reply:
Changed